include/triton/core/tritonbackend.h

```diff
@@ -973,6 +973,21 @@ TRITONBACKEND_DECLSPEC TRITONSERVER_Error*
 TRITONBACKEND_ModelAutoCompleteConfig(
     TRITONBACKEND_Model* model, bool* auto_complete_config);
 
+/// When using auto-complete for a model which supports batching, this value 
```

Comment:
Let's keep the API documentation as generic:

Get the default max batch size value for the model. When auto-completing the model configuration, the backend  may use this value to populate max_batch_size field.

\param model The model.
/// \param default_max_batch_size Returns the default_max_batch_size

Reply:
Updated to new comment